src/main/java/ch/wisv/areafiftylan/service/TicketServiceImpl.java

```diff
@@ -132,6 +132,13 @@ public void cancelTicketTransfer(String token){
         tttRepository.save(ttt);
     }
 
+    @Override
+    public Collection<TicketTransferToken> getTicketTransferTokensByUser(String username) {
```

Comment:
This is not unexpected behavior. Once the ticket is transferred, the user no longer has any control over this ticket. The user will not see the ticket because the TicketTransferToken is gone, also he is not able to set up a transfer. The fact that the TicketOwnerUsername changes is expected and correct behavior.